api/resources/messages.py

```diff
@@ -20,6 +20,22 @@ def _message_payload(message):
         'created_at' : message.created_at.__str__()
     }
 
+def _validate_field(data, field, proceed, errors, missing_okay=False):
+    # can't create a user if there is no unique email
+    # or if email/values/skills fields are empty
```

Comment:
these comments seem to be leftover from a different file since we're not creating a user here?